std/node/process.ts

```diff
@@ -0,0 +1,32 @@
+export const version = `v${Deno.version.deno}`;
+
+export const versions = {
+  node: Deno.version.deno,
+  ...Deno.version
+};
+
+const osToPlatform = (os: Deno.OperatingSystem): string =>
+  os === "win" ? "win32" : os === "mac" ? "darwin" : os;
+
+export const platform = osToPlatform(Deno.build.os);
+
+export const { arch } = Deno.build;
+
+export const argv = [Deno.execPath(), ...Deno.args];
+
+// TODO(rsp): currently setting env seems to be working by modifying the object
+// that is returnd by Deno.env(). Need to make sure that this is the final API
+// or Deno.env('key', 'value') is to be used in the future.
+export const env = Deno.env();
+
+export const { pid, cwd, chdir, exit } = Deno;
+
+export function on(_event: string, _callback: Function): void {
+  // TODO(rsp): to be implemented
```

Comment:
please add 
```
throw Error("unimplemented");
```

Reply:
Is it ok if I throw Error("unimplemented") for everything **except** "uncaughtException"? The idea is that Deno crashes on all uncaught exceptions anyway, and I want to prevent code that makes Node crash on uncaught exceptions to crash during adding a listener for uncaughtException.

Reply:
But it you think it's not a good idea then I can throw on everything and take care of it in a third party module, just let me know.

Reply:
Ok, I'm just throwing Error("unimplemented") for everything that is not actually implemented. I don't want to add any exceptions. Sorry for the confusion.